scene/main/viewport.cpp

```diff
@@ -162,6 +162,22 @@ ViewportTexture::~ViewportTexture() {
 }
 
 void Viewport::_sub_window_update_order() {
+	if (gui.sub_windows.is_empty())
```

Comment:
You can safely return here, if the size of the list is zero or one.

Reply:
Updated the commit